Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -154,8 +154,6 @@
 		E4BF2DEC1BACE92400DA9D68 /* ContentBlocker.appex */ = {isa = PBXFileReference; explicitFileType = "wrapper.app-extension"; includeInIndex = 0; path = ContentBlocker.appex; sourceTree = BUILT_PRODUCTS_DIR; };
 		E4BF2DF01BACE92400DA9D68 /* ActionRequestHandler.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = ActionRequestHandler.swift; sourceTree = "<group>"; };
 		E4BF2DF21BACE92400DA9D68 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
-		E4BF2DFA1BACEEDE00DA9D68 /* Blockzilla.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.xml; path = Blockzilla.entitlements; sourceTree = "<group>"; };
-		E4BF2DFB1BACEF1B00DA9D68 /* ContentBlocker.entitlements */ = {isa = PBXFileReference; lastKnownFileType = text.xml; path = ContentBlocker.entitlements; sourceTree = "<group>"; };
```

Comment:
I removed the .entitlement files because we don't need these anymore since the App Groups does this for us.